deploy/template.yaml

```diff
@@ -612,7 +612,7 @@ Resources:
                 - IsProduction
                 - "account.gov.uk"
                 - !If
-                  - IsNewDevelopment
+                  - IsDevelopment
                   - !Sub "${Environment}.dev.identity.account.gov.uk"
                   - !Sub "${Environment}.account.gov.uk"
```

Comment:
I think this might need to be updated to reflect the new dev domains - see https://github.com/alphagov/di-ipv-core-front/pull/1059/files#r1322787855